test/components/notice_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'component_test_helper'
+
+class NoticeTest < ComponentTestCase
+  def component_name
+    "notice"
+  end
+
+  test "fails to render when no parameters are given" do
```

Comment:
Current solution is that I've got the integration test housed in the component test - so now there's just one test file. It works, and I've commented the file to explain.